easybuild/easyconfigs/p/Perl/Perl-5.36.1-GCCcore-13.1.0.eb

```diff
@@ -0,0 +1,86 @@
+name = 'Perl'
+version = '5.36.1'
+
+homepage = 'https://www.perl.org/'
+description = """Larry Wall's Practical Extraction and Report Language
+
+Includes a small selection of extra CPAN packages for core functionality.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.1.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.cpan.org/src/%(version_major)s.0']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['68203665d8ece02988fc77dc92fccbb297a83a4bb4b8d07558442f978da54cc1']
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+
+dependencies = [
+    ('zlib', '1.2.13'),
+]
+
+# !! order of extensions is important !!
+# extensions updated on 2023-05-02
+# includes all dependencies for Autotools
```

Comment:
@Micket We'll need to wait until we've discussed this new "minimal extensions" approach before merging?

Reply:
Yes, though i strongly suspect this is the approach we have to go with more or less.